reduction/lr_reduction/time_resolved.py

```diff
@@ -124,7 +124,7 @@ def reduce_30Hz_slices(
         scan_index=scan_index,
         create_plot=create_plot,
         template_reference=template_reference,
-        q_summing=False,
+        q_summing=None,
```

Comment:
we should pass the value from the input parameter
```suggestion
        q_summing=q_summing,
```